### PR TITLE
Send `x-oasst-user` header to backend

### PR DESCRIPTION
**Pull request by AbdBarho**
#957 #953 

Also fix some UI errors regarding illegal nesting of html types.

### CODE REVIEW

File: website/src/components/Tasks/LabelTask/LabelTask.tsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import { Box, Button, Flex, HStack, Text, useColorModeValue } from "@chakra-ui/react";
-import { useEffect, useState } from "react";
+import { useEffect, useMemo, useState } from "react";
+import { useTranslation } from "react-i18next";
```
Comment (rjmacarthy) [inline website/src/components/Tasks/LabelTask/LabelTask.tsx:3]: Should it be `next-i18next`?

Comment (AbdBarho) [reply website/src/components/Tasks/LabelTask/LabelTask.tsx]: correct, this was leftover code from some experiment so I removed it.

**Review by rjmacarthy — APPROVED** [review]
LGTM

File: website/src/pages/api/admin/status.ts [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 import { withRole } from "src/lib/auth";
-import { oasstApiClient } from "src/lib/oasst_api_client";
-import { getBackendUserCore } from "src/lib/users";
+import { createApiClientFromUser } from "src/lib/oasst_client_factory";
 
 /**
  * Returns tasks availability, stats, and tree manager stats.
  */
 const handler = withRole("admin", async (req, res) => {
+  // NOTE: why are we using a dummy user here?
```
Comment (fozziethebeat) [inline website/src/pages/api/admin/status.ts:8]: This is to fetch global stats that apply to all users.  By fetching stats for the current admin user, the server would report stats that take into account what the admin user has done.  This is a hacky solution to simulate a global stats query.

**Review by fozziethebeat — APPROVED** [review]

**Pull request merged**